src/ngx_http_vhost_traffic_status_string.c

```diff
@@ -164,4 +164,66 @@ ngx_http_vhost_traffic_status_replace_strc(ngx_str_t *buf,
     return NGX_OK;
 }
 
+
+ngx_int_t
+ngx_hex_escape_invalid_utf8_char(ngx_pool_t *pool, ngx_str_t *buf, u_char *p, size_t n)
+{
+    u_char  c, *pb, *last, *prev;
+    size_t  len, size;
+    u_char   HEX_MAP[] = "0123456789ABCDEF";
+    uint32_t return_value;
+
+    last = p + n;
+    buf->data = ngx_pcalloc(pool, n * 5);
+
+    size = 0;
+    pb = buf->data;
+
+    for (len = 0; p < last; len++) {
+
+        if (*p < 0x80) {
+            if (*p == '"') {
```

Comment:
There's also `\` -> `\\` and `\n` -> `\\n` - https://github.com/prometheus/docs/blob/master/content/docs/instrumenting/exposition_formats.md#text-format-details

This appears to work: https://github.com/jongiddy/nginx-module-vts/commit/0d39f4d443a8b008c94d3c0ef56adea645e37043